core/src/main/java/com/kin/ecosystem/core/data/auth/AuthRepository.java

```diff
@@ -50,10 +53,9 @@ public static AuthRepository getInstance() {
 	}
 
 	@Override
-	public void setSignInData(@NonNull SignInData signInData) {
-		cachedSignInData = signInData;
-		localData.setSignInData(signInData);
-		remoteData.setSignInData(signInData);
+	public void setJWT(@NonNull String jwt) throws JSONException {
+		this.jwt = jwt;
```

Comment:
can we remove this.jwt and get jwt from localData

Reply:
not good form performance matters.